tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -58,6 +58,7 @@ static struct spi_dt_spec spi_slow = SPI_DT_SPEC_GET(SPI_SLOW_DEV, SPI_OP(FRAME_
 #define STACK_SIZE (512 + CONFIG_TEST_EXTRA_STACK_SIZE)
 #define BUF_SIZE 18
 #define BUF2_SIZE 36
+#define BUF3_SIZE 8192
```

Comment:
in your commit message: s/larget/large

That said, technically you are not really adding a new test. Just adding another buffer - certainly larger than the others - to testing multiple buffer tx/rx.

So add a new test, for large transfer and do not temper on existing ones.